tests/ludwig/hyperopt/test_hyperopt.py

```diff
@@ -86,3 +87,34 @@ def test_grid_search_more_than_one_sample():
             },
             num_samples=2,
         )
+
+
+def test_hyperopt_config_gbm():
```

Comment:
nit: it may be helpful to add a docstring referencing specific failures being tested for here, for our future reference